ddtrace/monkey.py

```diff
@@ -77,16 +77,28 @@ def patch(raise_errors=True, **patch_modules):
         >>> patch(psycopg=True, elasticsearch=True)
     """
     modules = [m for (m, should_patch) in patch_modules.items() if should_patch]
-    count = 0
     for module in modules:
-        patched = patch_module(module, raise_errors=raise_errors)
-        if patched:
-            count += 1
-
+        # TODO: this is a temporary hack until we shift to using
```

Comment:
Our next step is removing this hack and migrate all the instrumentation to this hook system.